app/templates/components/paper-autocomplete.hbs

```diff
@@ -58,15 +58,11 @@
 
         {{else}}
           {{#if showLoadingBar}}
-          {{!-- Render block template, then named component then default --}}
-            {{#if hasBlock}}
-                <li>{{yield searchText to="inverse"}}</li>
+          {{!-- Render named component then default --}}
+            {{#if notFoundComponent}}
+                <li>{{component notFoundComponent searchText=searchText}}</li>
             {{else}}
-              {{#if notFoundComponent}}
-                  <li>{{component notFoundComponent searchText=searchText}}</li>
-              {{else}}
-                  <li>{{notFoundMsg}}</li>
-              {{/if}}
+                <li>{{notFoundMsg}}</li>
             {{/if}}
```

Comment:
I think we can be a bit more friendlier and backwards compatible at the same time.
What do you think about:
```hbs
{{#if notFoundComponent}}
  <li>{{component notFoundComponent searchText=searchText}}</li>
{{else if (has-block "inverse")}}
  <li>{{yield to="inverse"}}</li>
{{else}}
  <li>{{notFoundMsg}}</li>
{{/if}}
```